worlds/ladx/__init__.py

```diff
@@ -459,51 +461,36 @@ def generate_output(self, output_directory: str):
                     loc.ladxr_item.location_owner = self.player
 
         rom_name = Rom.get_base_rom_path()

```

Comment:
I think there may be some sort of conflict here now because of the PR that was just merged to get the rom in `stage_assert_generate`, not sure tho. Either way your PR does have a conflict

Reply:
This PR
https://github.com/ArchipelagoMW/Archipelago/pull/3540

You reviewed it, so you'll know whether it's relevant to you

Reply:
Ack